models/projects.js

```diff
@@ -135,7 +135,33 @@ module.exports = {
             default : [] 
         },
         remarks : String,
-        userProfile : Object
+        userProfile : Object,
+        certificate : {
+            templateId : "ObjectId",
+            osid : {
+                type : String,
+                index : true,
+                unique : true
+            },
+            transactionId : {
+                type : String,
+                index : true,
+                unique : true
+            },
+            templateUrl : String,
+            status : String,
+            eligible : Boolean,
+            message : String,
+            issuedOn : Date,
+            criteria : Object,
+            originalTransactionInformation :{
```

Comment:
@VISHNUDAS-tunerlabs Please fix this object as discussed.

Reply:
@aks30 changed